doc/modules/model_evaluation.rst

```diff
@@ -2826,6 +2826,51 @@ Here are some usage examples of the :func:`d2_absolute_error_score` function::
 
 |details-end|
 
+|details-start|
+**D² log loss score**
```

Comment:
This is under „regression metrics“ but it’s a classification metric. Wen can fix that in another PR as it involves a larger change of the user guide.

Reply:
Haha you spot it as well. I assumed I would just merge it discretly and open an issue right after :smile: